aries-site/src/examples/templates/global-header/components/UserContext.js

```diff
@@ -0,0 +1,3 @@
+import React from 'react';
+
+export const UserContext = React.createContext(undefined);
```

Comment:
Should we establish somewhat of a template for what the context object should look like?

```suggestion
export const UserContext = React.createContext({
  user: {},
  setUser: () => {},
});
```

Reply:
Done.